src/theory/bv/theory_bv.cpp

```diff
@@ -861,9 +861,10 @@ bool TheoryBV::storePropagation(TNode literal, SubTheory subtheory)
   }
 
   // Propagate differs depending on the subtheory
-  // * bitblaster needs to be left alone until it's done, otherwise it doesn't know how to explain
+  // * bitblaster needs to be left alone until it's done, otherwise it doesn't
+  //   know how to explain
   // * equality engine can propagate eagerly
-  bool ok = true;
+  constexpr bool ok = true;
```

Comment:
Something doesn't quite make sense here. `ok` is always true, right? So the branch on line 870 is never taken. Shouldn't line 869 be `ok = d_out->propagate(literal)`?

Reply:
You can only use `constexpr` for `ok` if this is always true, right? If I address this issue in a future PR I will have to drop `constexpr` again? Maybe we should fix the underlying issue first. Is this PR urgent?

Reply:
> You can only use constexpr for ok if this is always true, right?

Correct.

> If I address this issue in a future PR I will have to drop constexpr again?

This seems like a trivial cost, and if I add a comment it falls to a few seconds to resolve by deleting it and the comment.

> Maybe we should fix the underlying issue first.

I suggest going forward as it is an improvement over the current state.

> Is this PR urgent?

No.

Reply:
I just wanted to make sure that this is what we have to do when we address this problem.
Yes, add a comment. Maybe open an issue and add the issue number. You can assign me.